src/lib/rest/StringFilter.cpp

```diff
@@ -1323,7 +1698,7 @@ bool StringFilter::match(ContextElementResponse* cerP)
 */
 StringFilter* StringFilter::clone(std::string* errorStringP)
 {
-  StringFilter* sfP = new StringFilter();
+  StringFilter* sfP = new StringFilter(type);
```

Comment:
Another case of `new StringFilter()` that has become in `new StringFilter(something)`. I wonder why...